### ISSUE TITLE
"X" button isn't closing the application

### ISSUE THREAD

**Issue by flamesjames** [devhubapp/devhub#272]
On the Windows Desktop application, I noticed that the "X" button in the top right of the window isn't working properly.

**Comment by KAUSTUBHDUBEY790** [comment]
i want to take on this issue, can you please elaborate

**Comment by DarshanDixit05** [comment]
Hello , I want to work on this issue . Can you please assign it to me ?

**Comment by flamesjames** [comment]
@brunolemos  - I am unable to assign this issue to folks.. would you be able to? Not sure if this is still an issue or not as I logged this awhile back..

**Comment by hiyorijl** [comment]
@flamesjames
> @brunolemos - I am unable to assign this issue to folks.. would you be able to? Not sure if this is still an issue or not as I logged this awhile back..

it's still an issue, close button doesn't work on Windows

**Comment by hiyorijl** [comment]
> @brunolemos - I am unable to assign this issue to folks.. would you be able to? Not sure if this is still an issue or not as I logged this awhile back..

@flamesjames here's a small tutorial to assign to other developers

1. you're already on the [issue page](https://github.com/devhubapp/devhub/issues/272)
2. Ubicate yourself under the title of this issue ' "X" button isn't closing the application #272' now on the right side of ityou have a green button saying New Issue, now **below that green button** there's **Assignees** 
3. In assigness there's the **button of settings**,  click on it, then you enter the name of a developer that is willing to work on this issue, in this case @DarshanDixit05 if he's still wants to.

take into consideration that you're **the only one** who can assign this issue to a developer since you're the owner of this issue

**Comment by Pooja1672** [comment]
Hello ,
Can you please assign this issue I would like to work on it

**Comment by flamesjames** [comment]
> > @brunolemos - I am unable to assign this issue to folks.. would you be able to? Not sure if this is still an issue or not as I logged this awhile back..
> 
> @flamesjames here's a small tutorial to assign this issue to other developers
> 
> 1. you're already on the [issue page](https://github.com/devhubapp/devhub/issues/272)
> 2. Ubicate yourself under the title of this issue ' "X" button isn't closing the application ["X" button isn't closing the application #272](https://github.com/devhubapp/devhub/issues/272)' now on the right side of ityou have a green button saying New Issue, now **below that green button** there's **Assignees**
> 3. In assigness just in the right side there's the **button of settings**,  click on it, then you enter the name of a developer that is willing to work on this issue, in this case @DarshanDixit05 if he's still wants to.
> 
> take into consideration that you're **the only one** who can assign this issue to a developer since you're the owner of this issue

Yes I know how to assign, but not getting the + button that's usually there. It just stays static as in the screenshot below:
<img width="205" alt="image" src="https://github.com/devhubapp/devhub/assets/40143723/6c1179e5-243f-4180-845f-8fada1b93d38">

Guessing I don't have proper permissions..

**Comment by DarshanDixit05** [comment]
Hey @flamesjames , I was tied up in some work , I want to work on it now and will fix it.
